src/components/Navbar/NavBar.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom';
+
+import NavBar from './NavBar';
+
+describe("NavBar", () => {
+  it('Should be true', () => {
+    expect(true).toBe(true);
+  });
+
+  it('Should display on the App', () => {
+    const mockFunction = jest.fn();
+    render(<NavBar currentUser={"Lucy"} signOut={mockFunction} />)
+    expect(screen.getByText('Lucy')).toBeInTheDocument();
+    expect(screen.getByText('Rancid Tomatillos')).toBeInTheDocument();
+  })
+
+  it("Should not display a user name if no user is logged in", () => {
```

Comment:
Excited to see, even just a simple test document, up and running given this is what we just learned!